spec/features/order_spec.rb

```diff
@@ -22,4 +20,11 @@
 
     expect(current_path).to eq orders_path
   end
+
+  scenario 'Publically display orders' do
+    order = create(:buy_order, quantity: 1337)
+
+    visit stock_path(order.stock)
+    expect(page).to have_content '1337'
```

Comment:
I would suggest you add a newline above the `expect`.

It's nice to have some visual separation between the arrange, act and assert sections of the spec.